File: C/ex19.c
//Exercicio 19


#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    // Declaração de variáveis
  float acel, vi, tp, v;

  // Solicita entrada de dados para o usuário e grava o valor digitado em uma variável "acel"
  printf("Digite a aceleracao em m/s^2: ");
  scanf("%f", &acel); 
  // Solicita entrada de dados para o usuário e grava o valor digitado em uma variável "vi"
  printf("Digite a velocidade inicial em m/s: ");
  scanf("%f", &vi); 
  // Solicita entrada de dados para o usuário e grava o valor digitado em uma variável "tp"
  printf("Digite o tempo do percurso em s: ");
  scanf("%f", &tp); 

  // Calculo da velocidade
  v=vi+(acel*tp);

  if(v<=40){
    printf("Veiculo Muito Lento");  
  }
   else if ( (40<v) && (v<=60) ){
      printf("Velocidade Permitida");   
   }
   else if ( (60<v) && (v<=80) ){
      printf("Velocidade de Cruzeiro");   
   }
   else if ( (80<v) && (v<=120) ){
      printf("Veiculo Rapido");   
   }
   else{
        printf("Veiculo Muito Rapido");   
   }
  


    return 0;
}

File: Python/ex19.py
#A partir dos valores da aceleração (a em m/s2), da velocidade inicial (v0 em m/s) e do tempo de percurso
#(t em s). Calcular e exibir a velocidade final de automóvel em km/h. Exibir mensagem de acordo com a
#tabela:

ac = float(input("Digite o valor da aceleracao:"))
vi = float(input("Digite o valor da velocidade inicial:"))
tempo = float(input("Digite o valor do tempo de percurso: "))

velocidade = vi+(ac*tempo)

print(f"A velocidade final do automovel é: {velocidade:.1f}")

if (velocidade <= 40):
    print("Veiculo muito lento")
elif (velocidade <= 60):
    print("Velocidade permitida")
elif (velocidade <= 80):
    print("Velocidade de cruzeiro")
elif (velocidade <= 120):
    print("Veiculo rapido")
elif (velocidade > 120):
    print("Veiculo muito rapido")